### PR TITLE
[MA-5658] Migrate monitor name in manifest to be aligned with monitor asset title

### PR DESCRIPTION
**Pull request by JoannaYe-Datadog**
### What does this PR do?

change manifest monitor titles to match titles specified in monitor asset file.

### Motivation

https://datadoghq.atlassian.net/browse/MA-5658

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: syncthing/manifest.json [hunk 1]
```diff
@@ -50,12 +50,12 @@
...
     "monitors": {
-      "[Syncthing] Disconnected": "assets/monitors/syncthing_disconnected.json",
-      "[Syncthing] Out of sync": "assets/monitors/syncthing_out_of_sync.json",
-      "[Syncthing] System error": "assets/monitors/syncthing_system_error.json",
-      "[Syncthing] Folder error": "assets/monitors/syncthing_folder_error.json",
-      "[Syncthing] Service error": "assets/monitors/syncthing_service_error.json",
-      "[Syncthing] Device not connected": "assets/monitors/syncthing_device_not_connected.json"
+      "No active connections": "assets/monitors/syncthing_disconnected.json",
+      "Files out of sync": "assets/monitors/syncthing_out_of_sync.json",
+      "System errorsr": "assets/monitors/syncthing_system_error.json",
```
Comment (drichards-87) [inline syncthing/manifest.json:55]:
```suggestion
      "System errors": "assets/monitors/syncthing_system_error.json",
```

File: syncthing/manifest.json [hunk 2]
```diff
@@ -50,12 +50,12 @@
...
+      "Folder errors": "assets/monitors/syncthing_folder_error.json",
+      "Service is failed": "assets/monitors/syncthing_service_error.json",
```
Comment (drichards-87) [inline syncthing/manifest.json:57]:
```suggestion
      "Service failed": "assets/monitors/syncthing_service_error.json",
```

**Review by drichards-87 — APPROVED** [review]
Left a couple of small suggestions from Docs and approved the PR.

**Review by yaodingyd — APPROVED** [review]

File: syncthing/manifest.json (hunk 2 above)
Comment (JoannaYe-Datadog) [reply syncthing/manifest.json:57]: I'll have to leave this as is to match the title in `syncthing/assets/monitors/syncthing_service_error.json`

**Review by moreauowen — APPROVED** [review]

**Review by alpinet — APPROVED** [review]

**Pull request merged**